specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2021-03-01/storagecache.json

```diff
@@ -1461,10 +1387,26 @@
             "type": "string"
           },
           "type": "array"
+        },
+        "dnsServers": {
+          "description": "DNS servers for the cache to use. The default is determined by the network configuration.",
```

Comment:
What does 'default is determined by the network configuration' mean? Say, if user doesn't provide dnsServers values during PUT, would the default values be returned by RP in GET? If so please be aware that PUT & GET mismatch and you will run into issues in What-If testing. The PUT should be as close as possible to GET payload.
Refer https://armwiki.azurewebsites.net/api_contracts/WhatIfNoiseFix.html#put-as-patch-properties and  https://armwiki.azurewebsites.net/api_contracts/WhatIfNoise.html

Reply:
I tried to reword this but I'm not sure how much else to add or how to phrase it.
With DNS, DHCP or AD or other services on the network can provide the DNS configuration.  In Azure I think virtual networks have a variety of options for setting up dns servers on virtual networks which could also provide the configuration.   Whatever the network provides will be sent back in a GET and can be supplied in a PUT again or left out of a PUT.  Either way would work.